lib/features.json

```diff
@@ -213,5 +213,10 @@
       "name": "hasAttackCooldown",
       "description": "If there is a cooldown after attacks to deal full damage.",
       "versions": ["1.9", "1.10", "1.11", "1.12", "1.13", "1.14", "1.15", "1.16"]
+    },
+    {
+      "name": "usesLoginPacket",
+      "description": "Uses the login packet as defined in mcData",
+      "versions": ["1.16.2", "1.16.3"]
```

Comment:
This should be for 1.16.1 too

Reply:
1.16.1 does not use the login packet.

Reply:
Or if it does, it's not included in mcData.

Reply:
Nevermind, I'm just dumb.